.github/ISSUE_TEMPLATE/icon-request.md

```diff
@@ -0,0 +1,24 @@
+name: Icon request
+about: Proposal for a new icon
+
+---
+
+## Summary
+
+_To help us better understand the purpose and role of icons, please use give us as much detail as possible. You may request an icon without specifying a specific shape, but please detail the purpose and use cases._
```

Comment:
"please provide as much detail as possible" instead of "please use give us as much detail as possible"

Reply:
I'm sorry. The suggested edit also had a typo in it...

"please give us as much detail as possible" instead of "please use give us as much detail as possible"